### PR TITLE
Ephemeral messages for groups

### PR DESCRIPTION
**Pull request by bennyn**
Can be tested on "edge" webapp

### CODE REVIEW

File: app/page/template/partials/template-message.htm [hunk 1]
```diff
@@ -1,4 +1,4 @@
-<script type="text/html" id="all-verified">
+  <script type="text/html" id="all-verified">
```
Comment (gregor) [inline app/page/template/partials/template-message.htm]: Discard change

File: app/page/template/partials/template-message.htm [hunk 2]
```diff
@@ -93,13 +94,13 @@
         <!-- /ko -->
       <!-- /ko -->
       <!-- ko if: asset.is_video() -->
-        <video-asset data-bind="css: {'bg-color-ephemeral': message.is_expired()}" params="asset: asset, expired: message.is_expired()"></video-asset>
+        <video-asset data-bind="css: {'bg-color-ephemeral icon-movie text-white flex-center': message.is_expired()}" params="asset: asset, expired: message.is_expired()"></video-asset>
```
Comment (gregor) [inline app/page/template/partials/template-message.htm:97]: Why do we do this here for audio/video assets but within the asset templace for files and link previews?

File: app/script/conversation/ConversationRepository.coffee [hunk 3]
```diff
@@ -1522,7 +1523,11 @@ class z.conversation.ConversationRepository
         else
           @logger.log 'Unsupported ephemeral type', message_et.type
     else
-      @delete_message_everyone conversation_et, message_et
+      if conversation_et.is_group()
+        user_ids = _.union [@user_repository.self().id], [message_et.from]
+        @delete_message_everyone conversation_et, message_et, user_ids
```
Comment (gregor) [inline app/script/conversation/ConversationRepository.coffee:1528]: Should optimally be sent to the sending client of the sender and not all of his clients.

File: app/style/fonts/zeta-neue.css [hunk 4]
```diff
@@ -252,3 +252,4 @@
 .icon-contacts:before {
   content: "\e602";
 }
+
```
Comment (lipis) [inline app/style/fonts/zeta-neue.css]: ?

File: app/script/conversation/ConversationRepository.coffee (hunk 3 above)
Comment (bennycode) [reply app/script/conversation/ConversationRepository.coffee:1528]: Will be done on the next iteration. Thanks! 👍

File: app/page/template/partials/template-message.htm (hunk 2 above)
Comment (bennycode) [reply app/page/template/partials/template-message.htm:97]: Has been approved by @herrmannplatz.

Comment (gregor) [reply app/page/template/partials/template-message.htm:97]: 👍

**Pull request merged**